README.md

```diff
@@ -13,20 +13,33 @@ platform][heroku] and lives here:
 
 ## Table of Contents
 
+- [Usale](#Usage)
 - [Deployment](#deployment)
 - [Running Halpy Locally](#running-halpy-locally)
 - [Configuration](#configuration)
 - [Scripting](#scripting)
 - [Notes](#notes)
 
+## Usage
+Halpy, like Hubot, understnads commands defined in indiivdual scripts. Thos scripts can be written by end users (like EDGI) or provided by external packages. Commands are issued by a user in a Slack channel who types `@halpy COMMANDNAME OPTIONS`.  So far we mostly use the commands we have written ourselves:
+
+- `scripts/zoom-scratch.coffee` provides several new commands for interacting with the Zoom video conferencing software. For the time being they are named `zoom test` instead of just `zoom`, in order to distinguish them from previous generations of the commands.  The new commands are: 
+  - `zoom test me now -t Meeting Topic` or `zoom test me [Date Descriptor] [for Time Descriptor] -t Meeting Topic`. Like the old `zoom me now` command, these commands create a new meeting, and recording can be turned on with either `-r` or `record` anywhere in the command. The exciting advance is that we can now provide a date for a future meeting, and we can also set the length of scheduled meetings with a time descriptor. Date descriptions are quite flexible, and take a vairety of forms such as "next Friday at 15:00", "September 15 at 3:00PM EDT," or "2019-09-15". ISO timestamps are also accepted. The time descriptor should take the form "for n minutes" or "for n hours", i.e., "for 120 minute" or "for 2 hours". The length `n` needs to be an integer so for 1.5 hr meetings please use `90 minutes` instead.
```

Comment:
Would instructions be simpler (especially in halpy's short help text), if we just said "uses minutes", rather than reference integers and hours at all?

Reply:
fixed in a1af09f  -- I updated the regex to accept decimal points!